config/config.go

```diff
@@ -147,20 +147,21 @@ type (
 		}
 
 		Google struct {
-			MachineType  string            `envconfig:"DRONE_GOOGLE_MACHINE_TYPE"`
-			MachineImage string            `envconfig:"DRONE_GOOGLE_MACHINE_IMAGE"`
-			Network      string            `envconfig:"DRONE_GOOGLE_NETWORK"`
-			Subnetwork   string            `envconfig:"DRONE_GOOGLE_SUBNETWORK"`
-			Labels       map[string]string `envconfig:"DRONE_GOOGLE_LABELS"`
-			Scopes       string            `envconfig:"DRONE_GOOGLE_SCOPES"`
-			DiskSize     int64             `envconfig:"DRONE_GOOGLE_DISK_SIZE"`
-			DiskType     string            `envconfig:"DRONE_GOOGLE_DISK_TYPE"`
-			Project      string            `envconfig:"DRONE_GOOGLE_PROJECT"`
-			PrivateIP    bool              `split_words:"true"`
-			Tags         []string          `envconfig:"DRONE_GOOGLE_TAGS"`
-			UserData     string            `envconfig:"DRONE_GOOGLE_USERDATA"`
-			UserDataFile string            `envconfig:"DRONE_GOOGLE_USERDATA_FILE"`
-			Zone         string            `envconfig:"DRONE_GOOGLE_ZONE"`
+			MachineType         string            `envconfig:"DRONE_GOOGLE_MACHINE_TYPE"`
+			MachineImage        string            `envconfig:"DRONE_GOOGLE_MACHINE_IMAGE"`
+			Network             string            `envconfig:"DRONE_GOOGLE_NETWORK"`
+			Subnetwork          string            `envconfig:"DRONE_GOOGLE_SUBNETWORK"`
+			Labels              map[string]string `envconfig:"DRONE_GOOGLE_LABELS"`
+			Scopes              string            `envconfig:"DRONE_GOOGLE_SCOPES"`
+			ServiceAccountEmail string            `envconfig:"DRONE_GOOGLE_SERVICE_ACCOUNT_EMAIL"`
+			DiskSize            int64             `envconfig:"DRONE_GOOGLE_DISK_SIZE"`
+			DiskType            string            `envconfig:"DRONE_GOOGLE_DISK_TYPE"`
+			Project             string            `envconfig:"DRONE_GOOGLE_PROJECT"`
+      PrivateIP           bool              `split_words:"true"`
```

Comment:
looks like an alignment issue. Go uses tabs instead of spaces. I recommend using the `gofmt` [1] utility to automatically format the code, which will ensure it conforms to style guides. If you are using an edit (like vscode) there is an official Go plugin that will automatically format when you save the file.

[1] https://blog.golang.org/gofmt